images/broker/runbroker-customize.sh

```diff
@@ -130,7 +130,7 @@ Xmx=$MAX_HEAP_SIZE
 Xmn=$HEAP_NEWSIZE
 MaxDirectMemorySize=$MAX_HEAP_SIZE
 # Set for `JAVA_OPT`.
-JAVA_OPT="${JAVA_OPT} -server -Xms${Xms} -Xmx${Xmx} -Xmn${Xmn}"
+JAVA_OPT="${JAVA_OPT} -server ${BROKER_MEM:-"-Xms${Xms} -Xmx${Xmx} -Xmn${Xmn}"} "
```

Comment:
Could you please describe more about this line? I am not sure how it works. The original script will calculate MEM according to system memory resource. And we are going to add resource limit in future version.

Reply:
BROKER_MEM will be set as an environment variable through configmap and we can use the given value first. If BROKER_MEM does not exist, we will use the calculated value.